srcpkgs/kdeplasma-addons/template

```diff
@@ -1,25 +1,28 @@
 # Template file for 'kdeplasma-addons'
 pkgname=kdeplasma-addons
-version=6.0.5
+version=6.1.0
 revision=1
 build_style=cmake
 build_helper=qemu
 configure_args="-DBUILD_TESTING=OFF -DKF6_HOST_TOOLING=/usr/lib/cmake
  -DKDE_INSTALL_QMLDIR=lib/qt6/qml -DKDE_INSTALL_QTPLUGINDIR=lib/qt6/plugins"
 hostmakedepends="extra-cmake-modules qt6-base qt6-tools pkg-config gettext
- qt6-declarative-host-tools
- kf6-kcmutils kf6-kpackage"
+ qt6-declarative-host-tools kf6-kcmutils kf6-kpackage"
 makedepends="kf6-kdeclarative-devel kf6-kholidays-devel kf6-krunner-devel
  kf6-sonnet-devel kf6-kunitconversion-devel kf6-knewstuff-devel
  plasma5support-devel qt6-qt5compat-devel kf6-kio-devel
  kf6-knotifications-devel kf6-kservice-devel kf6-kxmlgui-devel
- kf6-kcmutils-devel libplasma-devel"
+ kf6-kcmutils-devel libplasma-devel kf6-kauth-devel"
 short_desc="Various Plasma addons"
 maintainer="Luciogi <lucigithubcommit@skiff.com>"
 license="GPL-2.0-or-later, LGPL-2.1-or-later"
 homepage="https://invent.kde.org/plasma/kdeplasma-addons"
 distfiles="${KDE_SITE}/plasma/${version}/${pkgname}-${version}.tar.xz"
-checksum=911af4754873c17c1ca7d49f0669de3e386858b09adf282783b9c5510576ba95
+checksum=cb57fa427b3188a1bc7a6b12e196fc11252742e08f6397e3af00ffaf34676127
+
+if [ $CROSS_BUILD ];then
+	hostmakedepends+=" kf6-kauth-devel"
+fi
```

Comment:
No, put `kf6-kauth-tools` in `hostmakedepends` instead.

Reply:
Also add `qt6-webenegine-devel` on where applicable.

Reply:
I dont remember which pkgs need as optional dep qt6-webengine-devel

This can be done later